### PR TITLE
GitLab AWS Example

### PR DESCRIPTION
**Pull request by rbwsam**

### CODE REVIEW

File: cluster/examples/gitlab/aws/provider.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+apiVersion: aws.crossplane.io/v1alpha1
+kind: Provider
+metadata:
+  name: demo-aws
+  namespace: crossplane-system
+spec:
+  credentialsSecretRef:
+    key: credentials
+    name: demo-aws-creds
+  region: us-east-1
```
Comment (ichekrygin) [inline cluster/examples/gitlab/aws/provider.yaml]: Shouldn't this be REGION variable?

File: docs/gitlab/gitlab-aws.md [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+1. Click `Create DB Subnet Group`
+1. Name your subnet i.e. `eks-db-subnets`
+1. Select the VPC created in the EKS VPC step
+1. Click `Add all subnets related to this VPC`
+1. Click Create
+1. Export the db subnet group name
+    ```console
+    export RDS_SUBNET_GROUP_NAME=replace-with-DBSubnetgroup-name
+    ```
+#### Create an RDS Security Group (example only)
```
Comment (negz) [inline docs/gitlab/gitlab-aws.md]: Can we use the default security group of the VPC we created to put the EKS cluster in?

File: docs/gitlab/gitlab-aws.md [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+    export RDS_SECURITY_GROUP=replace-with-security-group-id
+    ```
+
+#### Create an Elasticache Subnet Group
+
+```console
+export REDIS_SUBNET_GROUP=replace-me-with-redis-subnet-group
+``` 
+
+#### Create a Redis security group
```
Comment (negz) [inline docs/gitlab/gitlab-aws.md]: Can we use the default security group of the VPC we put the EKS cluster in?

**Comment by bassam** [comment]
replaced by #482

**Pull request closed without merging**